spec/features/proposal/collection_draft_proposals/collection_draft_proposal_update_spec.rb

```diff
@@ -27,14 +27,40 @@
     end
 
     context 'when a proposal is not "in work"' do
-      before do
-        mock_submit(CollectionDraftProposal.first)
-        visit edit_collection_draft_proposal_path(@collection_draft_proposal)
+      context 'when trying to submit a proposal that is not "in work"' do
+        before do
+          mock_submit(CollectionDraftProposal.first)
+          visit edit_collection_draft_proposal_path(@collection_draft_proposal)
+        end
+
+        it 'cannot be edited' do
+          expect(page).to have_content('Only proposals in an "In Work" status can be edited.')
+          expect(page).to have_link('Cancel Proposal Submission')
+        end
       end
 
-      it 'cannot be edited' do
-        expect(page).to have_content('Only proposals in an "In Work" status can be edited.')
-        expect(page).to have_link('Cancel Proposal Submission')
+      context 'when trying to save updated data to a proposal that is not "in work"' do
+        # this context simulates the situation where a user has accessed the proposal and is editing data (typically using browser 'back')
+        # despite the proposal having already been submitted - proposal_status != 'in_work'
+        before do
+          visit edit_collection_draft_proposal_path(@collection_draft_proposal)
```

Comment:
Should use either `@collection_draft_proposal` or `CollectionDraftProposal.first` and not both.  I'm aware it was in the old tests, but you should update it while you're here.  The former is more independent.

Reply:
agree

Reply:
done